src/main/webapp/app/sentry/sentry.error-handler.ts

```diff
@@ -39,5 +39,8 @@ export class SentryErrorHandler implements ErrorHandler {
 
     handleError(error: any): void {
         captureException(error.originalError || error);
+        if (SentryErrorHandler.environment === 'local') {
+            throw error;
```

Comment:
Good catch! Sorry about that!
Maybe we should always throw the error? Let's see what @krusche thinks.

Reply:
Yes, we should probably always throw the error so that it will be visible in the users browser console. Otherwise it might be difficult to find problems on the production or test environments.

So, we simply remove the if statement here?

Reply:
fixed it